app/middlewares/schemas/users.js

```diff
@@ -55,3 +56,18 @@ exports.createUserSchema = [
     .withMessage(IS_STRING_ERROR),
   ...exports.signInSchema
 ];
+
+const queryParamsSchema = {
+  in: ['params', 'query'],
```

Comment:
Params or query ? why params?

Reply:
because in other companies it was requested to use both. Solved!